tests/inclusive_scan.cpp

```diff
@@ -48,10 +48,10 @@ void inclusive_scan_gold(std::vector<T>& v, T init = 0,
 
 // test the gold computation against a known value
 TEST_F(InclusiveScanAlgorithm, TestSTDInclusiveScan) {
-  std::vector<int> v = {5, 1, 6, 2, 6, 2, 5, 7};
+  std::vector<int> v    = {5, 1,  6,  2,  6,  2,  5,  7};
   std::vector<int> gold = {5, 6, 12, 14, 20, 22, 27, 34};
 
-  inclusive_scan_gold(v, 0, [](int a, int b) { return a + b; });
+  inclusive_scan_gold(v, 0, std::plus<int>());
```

Comment:
However, there is no guarantee that the std::plus<int> is valid SYCL device kernel code, and we have found that even trivial things like std::swap can be implemented in very weird ways in some platforms. To avoid introducing accidentally invalid SYCL code from a host library (e.g. system headers), I would rather revert this back to the lambda